src/scripting/level.cpp

```diff
@@ -22,6 +22,8 @@
 #include "supertux/game_session.hpp"
 #include "supertux/sector.hpp"
 
+#include "util/log.hpp"
```

Comment:
There shouldn't be a separation of includes here:

```suggestion
#include "util/log.hpp"
```

Reply:
Done.